.test-defs/CreateManagedSeed.yaml

```diff
@@ -16,4 +16,4 @@ spec:
     -shoot-name=$SHOOT_NAME
     -deploy-gardenlet=$DEPLOY_GARDENLET
     -backup-provider=$BACKUP_PROVIDER
-image: golang:1.16.2
+  image: golang:1.16.2
```

Comment:
In some environments we are hit by the docker hub rate limits.
It would be better to use an image from the GCR.
```suggestion
  image: eu.gcr.io/gardener-project/3rd/golang:1.16.2
```

Reply:
Sure, then I suppose it would be good to do the same for all test definitions?

Reply:
Until now we did not see any issues in the testmachinery with the docker.io images.
I think mostly because the node's image cache work great.

But to be safe for the future lets use our mirrored image for all tests.
@dguendisch any objections?

Reply:
We do run occasionally into docker hub rate limits, therefore Martin recently switched the images used by most/all(?) tests to gcr: https://github.com/gardener/gardener/pull/3785
 
> Sure, then I suppose it would be good to do the same for all test definitions?

Yes please.